### PR TITLE
Update settings.yml

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: settings.yml [hunk 1]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article19title: "HTML Background Color"
+article19link: "https://www.freecodecamp.org/news/html-background-color-change-bg-color-tutorial/"
+article20title: "Lowercase a String in JS"
+article20link: "https://www.freecodecamp.org/news/how-to-lowercase-a-string-in-javascript-tolowercase-in-js/"
+article21title: "Data Visualization Tools"
+article21link: "https://www.freecodecamp.org/news/data-visualization-tools-guide/"
+article22title: "CSS Selectors Cheatsheet"
+article22link: "https://freecodecamp.org/news/css-selectors-cheat-sheet-for-beginners/"
+article23title: "Sort Dict by Value Python"
+article23link: "https://www.freecodecamp.org/news/sort-dictionary-by-value-in-python/ "
```
Comment (scissorsneedfoodtoo) [inline settings.yml:89]: Minor issue, but there's a trailing space here. It's been fixed in the trending `english.yaml` file on the CDN, and should probably be removed here, too:

```suggestion
article23link: "https://www.freecodecamp.org/news/sort-dictionary-by-value-in-python/"
```

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
Just noticed one minor issue with a trailing space for one of the links.

All of the links and titles look good, so we should be able to get this in once the trailing space is removed.

**Comment by raisedadead** [comment]
Hi @ahmadabdolsaheb - please avoid using GitHub to edit these files. You can use GitPod if you are not feeling like cloning a repo.

It's easy to miss formatting issues. The last time we merged something on the CDN, it broke everything on the main repo. 

cc @ojeytonwilliams

**Comment by ahmadabdolsaheb** [comment]
Thanks for the heads up.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Took another look and everything LGTM 👍 👍

**Pull request merged**